index.html

```diff
@@ -0,0 +1,79 @@
+  <title></title>
+  <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.css">
+  <link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+
+  <div class="container">
+    <div class="row">
+      <div class="col-xl-12">
+        <div class="page-header text-center">
+          <h1>ROBOT RUMBLE</h1>
+          <h4>enter name and select robot type</h4>
+          <h4>then click "attack" to FIGHT!</h4>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-xs-6">
+        <div id="playerOne" class="playerOne">
+          <p>Player One:</p>
+          <input type="text" name="playerOne" class="input" id="playerOneName" autofocus>
+          <div>
+            <select name="select" id="selectRobotOne" class="form-control selectForm" disabled="true">
+              <option value="chooseOne">Choose a Robot</option>
+              <option value="stealthDrone" id="stealthDrone">Stealth Drone</option> 
+              <option value="tacoDrone" id="tacoDrone">Taco Drone</option>
+              <option value="stomperBipedal" id="stomperBipedal">Stomper Bipedal</option> 
+              <option value="looperBipedal" id="looperBipedal">Looper Bipedal</option> 
+              <option value="redneckATV" id="redneckATV">Redneck ATV</option>
+              <option value="scooterATV" id="scooterATV">Scooter ATV</option> 
+            </select>
+          </div> 
+          <div id="r1Div"></div>
+        </div>
+      </div>
+    <!-- </div> -->
+
+    <!-- <div class="row"> -->
+      <div class="col-xs-6">
+        <div id="playerTwo" class="playerTwo">
+          <p>Player Two:</p>
+          <input type="text" name="playerTwo" class="input" id="playerTwoName">
+          <div>
+            <select name="select" id="selectRobotTwo" class="form-control selectForm" disabled="true">
+              <option value="chooseOne">Choose a Robot</option>
+              <option value="stealthDrone" id="stealthDroneP2">Stealth Drone</option> 
+              <option value="tacoDrone" id="tacoDroneP2">Taco Drone</option>
+              <option value="stomperBipedal" id="stomperBipedalP2">Stomper Bipedal</option> 
+              <option value="looperBipedal" id="looperBipedalP2">Looper Bipedal</option> 
+              <option value="redneckATV" id="redneckATVP2">Redneck ATV</option>
+              <option value="scooterATV" id="scooterATVP2">Scooter ATV</option> 
+            </select>
+          </div>  
+          <div id="r2Div"></div>
+        </div>
+      </div>  
+    </div>  
+
+    <div class="row buttonDiv">
+      <div class="col-xl-12">
+        <div class="attackOutput text-center">
+          <button id="attackBtn" class="btn btn-lg btn-primary btn-danger" disabled="disabled">ATTACK!</button>
+          <div id="outputDiv"></div>
+        </div>
+      </div>  
+    </div>
+  </div>  
+
+  <script src="/node_modules/jquery/jquery.min.js" type="text/javascript"></script>
```

Comment:
This script tag didn't work for me. If it's working for you with the way that node has it installed in your project files, we should talk to Joe and see what's up.